.github/workflows/release_prep_hatch.yml

```diff
@@ -61,6 +64,11 @@ on:
         type: string
         required: false
         default: main
+      working-dir:
```

Comment:
Allow for working directory to be passed in so that `hatch` can reference the correct `pyproject.toml`.